andela/suyabay
# [#116873071] The Register Application button creates a new app with a JWT token

#### What does this PR do?

The Register Application button creates a new app with a JWT token
#### Description of Task to be completed?
- Created a table with 8 columns, including id, user_id, name, home_url, description, api_token,  
  created_at and updated_at using migration and model.
- Write a method that generates the JWT token. 
- Write a method that saves the information inputted on the form by the user using eloquent. This method 
  saves the data and returns the JWT token from the JWT token method. In the subsequent PR, the 
  JWT token will be stored in the database.
- The Register Application button create a new app with all the details and a suyabay JWT token that  
  has an expiry date of 30 days
- Write a method that displays all the apps created by the user.
- In this PR, I also minified api.main.css file, converted the api.main.css file to api.main.sass. Any  
  changes on the api.main.css file will be made on the  sass folder on api.main.sass.
#### How should this be manually tested?

Navigate to http://suyabay.app/developer/myapp/new which is a simple form. Fill the necessary field and hit 'SAVE CHANGES". This will create the app and display the generated JWT token. To test if the app has been successfully created, navigate to this url: http://suyabay.app/developer/myapp/appinfos. You should see the list of the created app.
#### Any background context you want to provide?

None
#### What are the relevant pivotal tracker stories?
- #116873071
#### Screenshots (if appropriate)

![screen shot 2016-04-06 at 8 26 45 pm](https://cloud.githubusercontent.com/assets/14159420/14329954/f42f8240-fc35-11e5-947d-5b5832b327ff.png)

This a sample of a generated JWT token.
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
